src/scale-manager/README.md

```diff
@@ -1,23 +1,115 @@
 
 ```javascript
 scaleManager.enable(function(resizeData) {
   console.log('The old listener is replaced with this one!', resizeData);
 });
 ```
+
+**Note:** There is a 500ms delay added to the event to solve an issues where on some browsers the window isn't fully resized before the event fires.
+
+### Callback Example
+
+The scale manager callback parameter contains `width`, `height`, and `scale`. these values can be used to resize your game. Depending on the engine you choose you'll have to use them differently. `scale` contains `x` and `y`, these are the independent scaling values for the x and y axis.
+
+```javascript
+scaleManager.enable(({width, height, scale}) => {
+  // -- PIXI -- //
+  const view = this.pixi.view;
+  const stage = this.pixi.stage;
+  const renderer = this.pixi.renderer;
+
+  stage.position.set(renderer.width / 2, renderer.height / 2);
+  stage.scale.set(scale.x, scale.y);
+
+  stage.pivot.x = renderer.width / 2;
+  stage.pivot.y = renderer.height / 2;
+
+  view.style.width = width + 'px';
+  view.style.height = height + 'px';
+
+  view.style.position = 'absolute'
+  view.style.left = '0px';
+  view.style.top = '0px'; 
+});
+```
+
+**Note**: A Phaser3 example can be found here: https://github.com/SpringRoll/Springroll-Seed/tree/templates/pixi
+
+## Safe Scaling
+Safe scaling is enabled by setting `safeWidth` and `safeHeight` in the scaleManager's config. 
+
+```javascript
+let scaleManager = new ScaleManager({
+  width: 1320,
+  height: 780,
+  safeWidth: 1024,
+  safeHeight: 660,
+  callback: pixiResize
+});
+```
+
+This allows for the scale manager to calculate the canvas's target size and scale to ensure that the viewport's minimum is never smaller than `safeWidth` and `safeHeight`, and It's maximum never larger than `width` and `height`.
+
+## Scaled Entities
+
+Any object that needs to be updated after a resize event can be implemented as a scaled Entity. The object must have an `onResize` function.
+
+```javascript
+class MyEntity extends ScaledEntity
+{
+  onResize({offset, scale, gameSize})
+  {
+    //Resize dependant code.
+  }
+}
+
+const myEntity = new MyEntity();
+
+scaleManager.addEntity(myEntity);
+scaleManager.removeEntity(myEntity);
+
+```
+
+| Parameter | values | Description |
+|---|---|---|
+|`offset`| `x,y` | distance calculated between the game size and the viewport size |
+|`scale`| `x,y ` | scale values calculated for viewport |
+|`gameSize`| `x,y` | actual game space size, set during scale manager initialization |
+
+### Anchor
+
+An Anchor is a scaled entity, provided by SpringRoll used too keep UI elements in a relative position regardless of the size of the viewport. for example if you always wanted to keep a health bar in the top left corner:
+
+```javascript
+const healthBar = new StatusBar("#ff0000"); // <-- not a springroll object.
+const healthAnchor = new Anchor({
+  position: { x: 66, y: 25 }, 
+  direction: { x: -1, y: -1 }, 
+  callback: (x, y) => healthBar.position.set(x, y)
```

Comment:
I'm not sure the `callback` here relates to `onResize` above in the `ScaledEntity` definition above.

Reply:
Anchors implement `onResize` do a little positional math based on the position/direction set by the user and the offset/gameSize from the resize event. Then it fires off the callback with the calculated x/y positions needed for the object.